packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -101,9 +104,12 @@ export default function CustomDataForm(props) {
           defaultValue=\\"johndoe@amplify.com\\"
           onChange={async (e) => {
             const { value } = e.target;
-            await runValidationTasks(\\"email\\", value);
+            if (errors.email?.hasError) {
```

Comment:
making sure I'm interpreting this correctly, onChange will only run validations if there is currently an error. If it briefly becomes valid, onChange validation will stop happening and will instead only validate once more during the onBlur event.

Reply:
Yes, that is correct